corgi/tasks/prod_defs.py

```diff
@@ -265,18 +265,29 @@ def parse_product_stream(
             "obj": product_stream,
         },
     )
-    if not stream_created:
-        _clean_orphaned_brew_tags(set(brew_tags_dict.keys()), name, str(product_stream.pk))
     parse_errata_info(errata_info, product, product_stream, product_stream_node, product_version)
+    if not stream_created:
```

Comment:
might be worth injecting some comments on rationale eg. this is always much faster then full deletion and reingest to make clear to our future selves.